app/src/main/res/values/strings.xml

```diff
@@ -5,6 +5,20 @@
     <!--Google APi-->
     <string name="default_web_client_id">154591189513-eilbr2jtms0av2ses4tsvo62009q5t7n.apps.googleusercontent.com</string>
 
+    <!-- Global -->
+    <string name="app_icon_content_description">HikeMate icon</string>
+    <string name="back_button_content_description">Back</string>
+
+    <!-- Fields -->
+    <string name="field_supportive_text_format">%d / %d</string>
+
+    <!-- Navigation -->
+    <string name="nav_map">Map</string>
+    <string name="nav_saved_hikes">Saved Hikes</string>
+    <string name="nav_profile">Profile</string>
+    <string name="nav_tutorial">Tutorial</string>
+    <string name="nav_auth">Auth(never used)</string>
```

Comment:
Question: Why have you put "(never used)" here? Wouldn't it be better to not put it, or to remove the string, because if someone use it, it would have an unpleasant surprise...

Reply:
As you wish. This was done as this is used for creating a `TopLevelDestination` which require a `textId`.
I remove the "(never used)" to make sure we don't have surprises in the future